lib/Doctrine/Migrations/InlineParameterFormatter.php

```diff
@@ -64,14 +65,14 @@ private function formatParameter(mixed $value, string|int $type): string|int|nul
         return $this->parameterToString($value);
     }
 
-    /** @param int[]|bool[]|string[]|array|int|string|bool $value */
-    private function parameterToString(array|int|string|bool $value): string
+    /** @param int[]|bool[]|string[]|float[]|array|int|string|float|bool $value */
+    private function parameterToString(array|int|string|float|bool $value): string
     {
         if (is_array($value)) {
             return implode(', ', array_map($this->parameterToString(...), $value));
         }
 
-        if (is_int($value) || is_string($value)) {
+        if (is_int($value) || is_string($value) || is_float($value)) {
             return (string) $value;
```

Comment:
Why did a string value need a string type cast?

Reply:
It doesn't, it's a cheap no-op. I suppose the original author simply wanted to keep the code short.